interface/basisu_c_binding/inc/basisu_c_binding.h

```diff
@@ -3,11 +3,11 @@
 
 #pragma once
 
-#ifdef DLL_EXPORT_FLAG
+#if defined(_MSC_VER)
 #define DLL_EXPORT __declspec(dllexport)
```

Comment:
This is going to be `__declspec(dllexport)` for clients trying to link to the resulting library too (if they `#include "basisu_c_binding" to get the function declarations)  so links will fail. clients need `__declspec(dllimport)`. What am I missing? How are the transcodetests building?

Reply:
Right!

transcodetests link against `obj_basisu_cbind`, which is an `OBJECT` type library, so it might works because of that.

This is subject to change in the [apple silicon PR](https://github.com/KhronosGroup/KTX-Software/pull/415), so we probably should do that one first.

I'll look into it, but making a dedicated compile option (similar to `KTX_API`) is probably a more robust solution.

Reply:
I've landed the Apple Silocon PR. Please take a look at this issue.

Reply:
Please fix this. It is still always defining `__declspec(export)`. The usual way to do this is what we are doing with `KTX_API` in `ktx.h`. For _ MSC_VER, unless already defined, it is defined as `__declspec(dllimport)`. When compiling the library the compile commands define it via `-DKTX_API=__declspec(dllexport)`. That way users don't need to do anything to ensure apps are compiled with `dlimport`.

Reply:
I introduced `KTX_BASISU_API`, which is similar to `KTX_API`